evutil.c

```diff
@@ -358,7 +358,7 @@ evutil_fast_socket_nonblocking(evutil_socket_t fd)
 int
 evutil_make_listen_socket_reuseable(evutil_socket_t sock)
 {
-#ifndef _WIN32
+#ifdef SO_REUSEADDR
```

Comment:
The behavior of this flag on win32 differs from unix, do you mind if I will change this to?
``` #if defined(SO_REUSEADDR) && !defined(_WIN32) ```